nilearn/interfaces/fmriprep/load_confounds_utils.py

```diff
@@ -35,7 +35,26 @@
 
 
 def _check_params(confounds_raw, params):
-    """Check that specified parameters can be found in the confounds."""
+    """Check that specified parameters can be found in the confounds.
+
+    Used for motion, wm_csf, global_signal, and compcor regressors.
+
+    Parameters
+    ----------
+    confounds_raw : pandas.DataFrame
+        Raw confounds loaded from the confounds file.
+
+    params : list of str
+        List of parameters constructed based on users choices.
+
+    Returns
+    -------
+    bool
+        True if all parameters are found in the confounds.
+        False if none of the parameters are found in the confounds.
+        list of str
+            List of parameters that are not found in the confounds.
```

Comment:
Something went wrong here. Maybe something like:
```suggestion
    bool or list of str
        True if all parameters are found in the confounds.
        False if none of the parameters are found in the confounds.
        List of parameters that are not found in the confounds
        if only some parameters are found.
```